### PR TITLE
header animations only on hover; fixes #23

### LINKED ISSUE #23: save my poor battery
I love animated logos; it looks terrific on the homepage!  Could we maybe limit the animation in the title bar on docs pages to just moving on hover or something?  I've been sitting here studying, trying to pick up Preact, when my laptop fan starts humming.  I'm on battery, so that's kind of frustrating...

I'm on a newer laptop, quad-core i3, so I'm sure this is even worse on some of the low-power chromebooks or something.

What do y'all think?

### PR DESCRIPTION
**Pull request by robbiewxyz**
This solves #23 by pausing animations in the header except when the mouse is hovering over that menu item.

### CODE REVIEW

File: src/components/header/index.js [hunk 1]
```diff
@@ -101,15 +101,21 @@ class NavItem extends Component {
...
+class NavLink extends Component {
+	state = { hovered: false }
+
+	render({ to, ...props }, { hovered }) {
+		let LinkImpl = to.path ? Link : 'a';
+		let Flair = to.flair && LINK_FLAIR[to.flair] && LINK_FLAIR[to.flair]
+		return (
+			<LinkImpl href={to.path || 'javascript:'} {...props} onMouseover={ () => this.setState ({ hovered: true }) }
+			                                                     onMouseout ={ () => this.setState ({ hovered: false }) }>
+				<Flair paused={!hovered} />
```
Comment (developit) [inline src/components/header/index.js]: I think this might need to check if `Flair` is defined:

```js
{ Flair && <Flair paused={!hover} /> }
```

**Comment by developit** [comment]
Looks good! Just a that one nitpicky thing :) I don't mind patching that in after merge, just thought it might be helpful to leave the note. Ready to merge?

**Comment by robbiewxyz** [comment]
I got the code checking in case that link doesn't have a Flair.

**Comment by developit** [comment]
Great! I'll merge this ASAP

**Pull request merged**